core/capabilities/ccip/ccipsolana/executecodec.go

```diff
@@ -26,6 +26,13 @@ func NewExecutePluginCodecV1() *ExecutePluginCodecV1 {
 }
 
 func (e *ExecutePluginCodecV1) Encode(ctx context.Context, report cciptypes.ExecutePluginReport) ([]byte, error) {
+	if len(report.ChainReports) == 0 {
```

Comment:
Same kind of change is needed on the ccipsolana/commitcodec.go file too right?
cc @huangzhen1997